fp-topics/hendelser-behandling/src/main/java/no/nav/vedtak/hendelser/behandling/BehandlingHendelse.java

```diff
@@ -22,5 +23,6 @@ public abstract class BehandlingHendelse {
     public abstract Kildesystem getKildesystem();
 
     public abstract Hendelse getHendelse();
+    public abstract LocalDateTime getTidspunkt();
```

Comment:
Trenger du den her? ref abakus - kontrakt-vedtak/src/main/java/no/nav/abakus/vedtak/ytelse/Ytelse.java

Reply:
La det her for å force fremtidige versjoner å inneholde tidspunkt